commit 7217ea9cf297e664be876eb82b66a1b4f157ca80
Author: Peter Seiderer <ps.report@gmx.net>
Date:   Fri Apr 3 21:52:30 2020 +0200

    fix libdrm related configure test failure
    
    Fixes:
    
      Project ERROR: Test config.qtwayland_client.tests.dmabuf-server-buffer tries to use undeclared library 'drm'
      Project ERROR: Test config.qtwayland_compositor.tests.dmabuf-server-buffer tries to use undeclared library 'drm'
      Project ERROR: Test config.qtwayland_compositor.tests.dmabuf-client-buffer tries to use undeclared library 'drm'
    
    Task-number: QTBUG-83303
    Change-Id: I3970f4e3855fcfa760493631a14adb6a16950aa9
    Reviewed-by: Ville Voutilainen <ville.voutilainen@qt.io>
    Reviewed-by: Heikki Halmet <heikki.halmet@qt.io>
    Reviewed-by: Eskil Abrahamsen Blomfeldt <eskil.abrahamsen-blomfeldt@qt.io>
    (cherry picked from commit ca63fe81df8ac910f43539719566ed6745700c2b)
    (cherry picked from commit 1344ca6b5a8fbc1b28727feea6d856af8bf3bc93)

diff --git a/src/client/configure.json b/src/client/configure.json
index 06213968..27c73f89 100644
--- a/src/client/configure.json
+++ b/src/client/configure.json
@@ -74,7 +74,8 @@
             "label": "Linux dma-buf Buffer Sharing",
             "type": "compile",
             "test": "dmabuf_server_buffer",
-            "use": "egl drm"
+            "libs": "-ldrm",
+            "use": "egl"
         },
         "vulkan-server-buffer": {
             "label": "Vulkan Buffer Sharing",
diff --git a/src/compositor/configure.json b/src/compositor/configure.json
index 0dbb3364..73e4c1a8 100644
--- a/src/compositor/configure.json
+++ b/src/compositor/configure.json
@@ -80,13 +80,15 @@
             "label": "Linux dma-buf Buffer Sharing",
             "type": "compile",
             "test": "dmabuf_server_buffer",
-            "use": "egl drm"
+            "libs": "-ldrm",
+            "use": "egl"
         },
         "dmabuf-client-buffer": {
             "label": "Linux Client dma-buf Buffer Sharing",
             "type": "compile",
             "test": "dmabuf_client_buffer",
-            "use": "egl drm"
+            "libs": "-ldrm",
+            "use": "egl"
         },
         "vulkan-server-buffer": {
             "label": "Vulkan Buffer Sharing",
